aiidalab_widgets_base/viewers.py

```diff
@@ -153,11 +154,11 @@ def __init__(self, parameter, downloadable=True, **kwargs):
 
 
 class _StructureDataBaseViewer(ipw.VBox):
-    """Base viewer class for AiiDA structure or trajectory objects.
+    """Base viewer class for AiiDA structure or ase structure objects.
 
     :param configure_view: If True, add configuration tabs (deprecated)
     :type configure_view: bool
-    :param configuration_tabs: List of configuration tabs (default: ["Selection", "Appearance", "Cell", "Download"])
+    :param configuration_tabs: List of configuration tabs (default: ["Cell", "Selection", "Cell", "Download"])
```

Comment:
What are the allowed values here? Should we maybe provide an `Enum` of valid values?

Reply:
These are four allowed values. I think `Enum` makes it more robust, will give it a try.

Reply:
"Cell" appears twice.

Reply:
As we discussed in the meeting, not necessary change to `Enum` the scope of these options are all only used in this function for mapping.